sdk/keyvault/keyvault-keys/migration-guide.md

```diff
@@ -0,0 +1,251 @@
+# Guide for migrating to @azure/keyvault-keys from azure-keyvault
+
+This guide is intended to assist in the migration to `@azure/keyvault-keys` from `azure-keyvault`. It will focus on side-by-side comparisons for similar operations between the two packages.
+
+Familiarity with the `azure-keyvault` package is assumed. For those new to the Key Vault client libraries for JavaScript, please refer to the [README for name of @azure/keyvault-keys here][kvk-readme] rather than this guide.
```

Comment:
This is the result of my previous comment. You got to read the suggested edits before applying them :)

```suggestion
Familiarity with the `azure-keyvault` package is assumed. For those new to the Key Vault client libraries for JavaScript, please refer to the [README for @azure/keyvault-keys][kvk-readme] rather than this guide.
```

Reply:
Thanks!